tests/integration/API/Pages/Read/ResponseTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Tests\API\Pages\Read;
+
+use SkyVerge\WooCommerce\Facebook\API\Pages\Read\Response;
+
+/**
+ * Tests the API\Pages\Read\Response class.
+ */
+class ResponseTest extends \Codeception\TestCase\WPTestCase {
+
+
+	/** @var \IntegrationTester */
+	protected $tester;
+
+
+	public function _before() {
+
+		parent::_before();
+
+		require_once 'includes/API/Response.php';
+		require_once 'includes/API/Pages/Read/Response.php';
+	}
+
+
+	/** Test methods **************************************************************************************************/
+
+
+	/** @see Response::get_name() */
+	public function test_get_name() {
+
+		$raw_response = json_encode( [ 'name' => 'Test Page', 'link' => 'https://example.org' ] );
```

Comment:
We should probably test with invalid response data as well.

Reply:
Yes, that sounds good. Even in the case of an API error, the response object is created with a JSON object like the one [here](https://developers.facebook.com/docs/graph-api/using-graph-api/error-handling#handling-errors), so we could test with that and confirm that the method return `null` (instead of a notice).